### PR TITLE
Milestone 1

### PR DESCRIPTION
**Pull request by nwakauc**
## Inserting and querying data in  `animals` table

- [ ] I used the given repo to generate my own repo,
- [ ] I downloaded and installed PostgreSQL,
- [ ] I created a database named `vet_clinicdb`,
- [ ] I created a table named animals with columns.
- [ ] I inserted data into the database
- [ ] And implemented the suggested changes from the code reviewer.

## 📸 Screenshots of query results
db
db1

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -1,5 +1,18 @@
...
+-- Table: public.animals
 
-CREATE TABLE animals (
-    name varchar(100)
-);
+-- DROP TABLE IF EXISTS public.animals;
+
+CREATE TABLE IF NOT EXISTS public.animals
+(
+    id integer,
```
Comment (rotimiazeez) [inline schema.sql]: - [ OPTIONAL ] Please note that you make the id column not null and autoincremented by changing you existing code to:
```
id int NOT NULL IDENTITY(1,1)
```

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi @uchexm  👍🏽,
## Good job so far!
### To Highlight 👇🏽
- No Linter error ✔️ 
- Good PR description 👍🏽 
- Good commit message(s) 😉
- Descriptive README 📖
You still need to work on some issues to prepare your project for the final evaluation, but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR WORDS>


## Suggested changes
## Please ensure you add the screenshots of your query results to the pull request description. 🙏 
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you consider them as they can improve your code._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag (@rotimiazeez) me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

File: schema.sql (hunk 1 above)
Comment (nwakauc) [reply schema.sql]: Done

**Review by muneebulrehman — APPROVED** [review]
Hi @uchexm , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🥳
Congratulations! 🎉 

# Status: Approved✔️

## Highlights
- Descriptive PR
- Good follow-up with the previous reviewer.

## Optional suggestions

- [ ] Kindly add a section **Future Features** to your README file describing 1-3 potential features to be added in the future.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**